modules/ROOT/pages/hardening-your-mule-installation.adoc

```diff
@@ -4,16 +4,13 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: fine tuning, configuration, security
 
-* <<Hardening Checklist>>
-* <<Hardening in Layers>>
+As distinct from Security, hardening refers to the steps one must perform in order to bring an application from development into production. These are the hoops one must jump through in order to deploy something in the IT space after you complete the development cycle.
 
-As distinct from Security, hardening refers to the steps one must perform in order to bring an application from development into production. These are the hoops one must jump through in order to get something deployed in the IT space after development is completed.
-
-Normally, an IT organization already has control of keeping open ports to a minimum, restricting access to administrative applications, minimizing the number of applications, and other housekeeping. Of course, the files used to configure Mule should be as secure as any configuration files.
+Normally, an IT organization already has control of keeping open ports to a minimum, restricting access to administrative applications, minimizing the number of applications, and other housekeeping. Of course, use secure files to configure Mule as any configuration files.
```

Comment:
"and other housekeeping" > "and other measures to ensure quality"

"Of course, use secure files to configure Mule as any configuration files." > "We recommend using secure files to configure Mule, as well as using secure configuration files."